modules/suncalc.js

```diff
@@ -279,7 +279,7 @@ function hoursLater(date, h) {
 // calculations for moon rise/set times are based on http://www.stargazing.net/kepler/moonrise.html article
 
 SunCalc.getMoonTimes = function (date, lat, lng, inUTC) {
-    var t = new Date(date);
+    var t = typeof(date) === "object" ? date : new Date(date);
```

Comment:
Espruino does not seem to support the
new Date(object) constructor

Reply:
Thanks for spotting that! I thought I'd tried to diff most of the clones of this file to see if there were differences, but I must have missed this one